yaml/water.yaml

```diff
@@ -18,8 +18,8 @@ globals:
   - &water_standard_min_zoom
     case:
       - when: { way_area: { min: 102400 } }
-        then: 9
-      - else: { clamp: { min: 9, max: 16, value: { sum: [ { col: zoom }, 1.066 ] } } }
+        then: 8
```

Comment:
For this and all other changes in this file...

To match up with Natural Earth changes I'd expect this to be a clamp with min of 0 which is when we want to introduce any water layer content.

Then we need to pair it with a zoom transition in the OpenStreetMap JINJA logic for zoom 8.

```
        then: 0
      - else: { clamp: { min: 0, max: 16, value: { sum: [ { col: zoom }, 1.066 ] } } }
```